src/sentry/models/email.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
+
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+from sentry.db.models import FlexibleForeignKey, Model, sane_repr
+from sentry.utils.http import absolute_uri
+
+
+class Email(Model):
```

Comment:
I'd like to see this model named `UserEmail`. `Email` is too ambiguous and generic. If we wanted to start logging actual emails in the database, or something like that, this would be coming awkward. `UserEmail` is a more clear name that reflects what it is imo.